.cspell.json

```diff
@@ -812,6 +820,9 @@
     "midnightblue",
     "milee",
     "mintcream",
+    "minx",
+    "miny",
+    "minz",
```

Comment:
These should be `{max,min}_{x,y,z}`.

Reply:
They are written in `.xml` file.
I think it is better not to change those files.
So I changed ignoring `"(max|min)(xyz)"` in .xml files.

Reply:
Thank you! Added another comment. https://github.com/tier4/autoware-spell-check-dict/pull/280#discussion_r880419583